packages/core/src/providers/starknet.tsx

```diff
@@ -141,19 +141,25 @@ function useStarknetManager({
 
   const { account, library, error } = state
 
-  const connect = useCallback((connector: Connector) => {
-    connector.connect().then(
-      (account) => {
-        dispatch({ type: 'set_account', account: account.address })
-        dispatch({ type: 'set_provider', provider: account })
-        dispatch({ type: 'set_connector', connector })
-      },
-      (err) => {
-        console.error(err)
-        dispatch({ type: 'set_error', error: new ConnectorNotFoundError() })
-      }
-    )
-  }, [])
+  const connect = useCallback(
+    (connector: Connector) => {
```

Comment:
Looks like there is no formatter config file, like a simple `.prettierrc.json` for example, I think it could be great 👍🏽

Reply:
Prettier is configured in [package.json](https://github.com/apibara/starknet-react/blob/main/package.json#L50), it runs automatically when you run `pnpm lint:fix`. Formatting is also checked as part of the linting stage.

Reply:
Ok ! My bad 👍🏽